spec/presenters/summary/html_sections/children_details_spec.rb

```diff
@@ -79,7 +93,7 @@ module Summary
         expect(answers[5]).to be_an_instance_of(MultiAnswer)
         expect(answers[5].question).to eq(:child_respondents_relationship)
         expect(answers[5].value).to eq(['mother'])
-        expect(c100_application).to have_received(:respondents)
+        expect(c100_application).to have_received(:respondents).at_least(:once)
```

Comment:
Can we also test the change_paths? Specially here is important as there are many subtleties with paths.
Take a look at https://github.com/ministryofjustice/c100-application/blob/master/spec/presenters/summary/html_sections/abduction_spec.rb#L54 for example.